website/content/docs/getting-started/getting-started-with-k0s-and-terraform.md

```diff
@@ -30,7 +30,7 @@ cluster_flavor = "m5.large"
 region = "us-east-1"
 ```
 2. `terraform init`
-3. `terraform apply -auto-approve`
+3. `terraform apply -auto-approve --var-file="terraform.tfvars.example"`
```

Comment:
Actually, the instructions should tell the user to copy `terraform.tfvars.example` to `terraform.tfvars` and modify the variables as needed. This is convention that I see in with most terraform code.

Reply:
Generally, the terraform scripts provide `terraform.tfvars.example` file and ask users to use as a guide to create `terraform.tfvars` file.

Reply:
Changed the instructions to match instead 👍